src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -33,6 +34,7 @@
 	private long executionTimeout = 5_000;
 	private String fromFileClass;
 	private Boolean opaqueTypes;
+	private Boolean noVersionCheck;
```

Comment:
Why you use object rather than primitive for this field?
Do you really want to process all three states (`null`, `true`, `false`) or two of them will be enough?